korangar/Cargo.toml

```
@@ -37,6 +37,7 @@ rayon = { workspace = true }
 ron = { workspace = true }
 serde = { workspace = true }
 sevenz-rust2 = { workspace = true, features = ["compress", "zstd"] }
+smallvec =  { workspace = true }
```

```suggestion
smallvec = { workspace = true, features = ["const_generics", "const_new", "union"] }
```

This makes SmallVec simpler to use and more efficient. You also have access to "const_new()" then.